src/main/resources/db/research/V100_0__service_providers.sql

```diff
@@ -0,0 +1,9 @@
+insert into service_provider
```

Comment:
_observation_
If these `V100` version numbers are causing migration problems on deployment, we can avoid them by marking them repeatable (naming it `R__something.sql`), which will not disturb the versioning and possibly doesn't require a separate deployment.

Reply:
oh interesting i'll look more into this, thanks